logging/src/Logging4cplus.cpp

```diff
@@ -24,7 +24,7 @@ using namespace log4cplus;
 
 namespace logging {
 
-static const char* logPattern = "%X{rid}|%d{%m-%d-%Y %H:%M:%S.%q}|%-5p|%c|%X{cid}|%X{sn}|%X{thread}|%b:%L|%M|%m%n";
+static const char* logPattern = "%d{%Y-%m-%dT%H:%M:%S,%qZ}|%-5p|%X{rid}|%c|%X{thread}|%X{cid}|%X{sn}|%b:%L|%M|%m%n";
```

Comment:
Note: this only a default format for ConsoleAppender.
Actual runtime configuration is in the configuration file.

Reply:
Thanks for pointing this out. Yes, this is intentional but when I was playing around with logging w/o a config file I wanted to see the same output :)